mkdocs.yml

```diff
@@ -64,7 +64,16 @@ markdown_extensions:
   - toc:
       permalink: true
 nav:
-  - Introduction: index.md
+  - Getting Started:
+    - Introduction: README.md
```

Comment:
hold this change, its conflicting with https://github.com/ovn-org/ovn-kubernetes/pull/4333/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2 why is the index.md file getting removed like that?

Reply:
I renamed index.md to README.md so we could both include it in the nav bar and use it as the homepage